docs/lib.rst

```diff
@@ -0,0 +1,6 @@
+Library
```

Comment:
Is there any way to have these .rst files generated by Sphinx somehow?  I feel that this is essentially what is happening when we call the following command:
`sphinx-apidoc -f -e -d 4 -o ./docs ./honeybee_energy`
I guess adding a new .rst file every time that we add a new CLI group isn't that bad but it would be good to make sure there are no automated options for this since we will definitely be adding more groups in the future and (probably) forgetting to add an rst file here.

Reply:
On a related note, does this mean that the command we have to run to generate the docs is now:
```
sphinx-apidoc -f -e -d 4 -o ./docs/api ./honeybee_energy
```
If so, can you update the ReadMe.MD to reflect this?

Reply:
I agree, you nailed it! It would be great to auto generating the all the cli rst files for each group. So the bad news is that it is not as straightforward as the sphinx -e option that separates modules, nothing is clearly available as an option within the sphinx-cli extension at least, 
That said, I'm not ready to give up, I will like to look into it deeper if that's okay. If a simple approach is not possible we can look at less straightforward ways to get there. I guess the last resource would be to try to generate these files with a routine inside the conf.py (can only try!).
 
On your second question, the command should stay as usual (without the /api), Also we can eliminate the api.rst file if better (see below).

Reply:
Sounds good and let me know if you find anything. Manually adding the `.rst` files is definitely not the end of the world and it's an acceptable solution if it's the best that we find. Still, if we are going to deploy a set of practices across several repos, it seems worthwhile to invest some time up-front in figuring out what the best setup is.

Reply:
I agree, will focus on this and will keep you posted!